server/repository/src/db_diesel/activity_log_row.rs

```diff
@@ -58,6 +58,7 @@ pub enum ActivityLogType {
     PrescriptionStatusPicked,
     PrescriptionStatusVerified,
     SensorLocationChanged,
+    ZeroQuantitiesForInvoiceCreatedBeforeStore,
```

Comment:
The "CreatedBeforeStore" part should be removed here and in the migration... (if we decide that we actually need a new endpoint, see other comment...)